.ci/scripts/generate_eol.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# This is a wrapper script to encapsulate calls like the following:
+# docker run -v \"$(pwd)\":/src -t update_eol_doc --release {$env.RELEASE_VER} --doc /src/docs/upgrading.asciidoc > ./docs/upgrading.asciidoc"
+
+# The mount point internal to the Docker container where we want to operate on this repo
+DOCKER_MOUNT_POINT=/src
+
+# The upgrading file, relative to the root of the repo.
+UPGRADING=/docs/upgrading.asciidoc
+
+# The tag designating the container that was built from the `generate_cc_changelog` project
+TAG=update_eol_doc
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    -v|--version)
+    VERSION="$2"
+    shift
+    shift
+    ;;
+    -h|--help)
+    echo "Use: generate_eol.sh --version 0.8.9"
+    exit 0
+    ;;
+    *)    # unknown option
+    POSITIONAL+=("$1") # save it in an array for later
+    shift # past argument
+    ;;
+esac
+done
+
+docker run \
+-v "$(pwd)":${DOCKER_MOUNT_POINT} \
```

Comment:
is it required to run with the flags `-t` or `-ti`?

Reply:
The `-t` flag is appended [on L39](https://github.com/elastic/apm-agent-nodejs/pull/1714/files/f7171b04a30361f5101edb2ff3c8d0fad190ea48#diff-ea60c515fa2f2c7eefd57bb0912f4491R39)

Reply:
I meant `--tty `, what's `${TAG}` required for?

Reply:
see https://github.com/elastic/apm-agent-nodejs/pull/1714/files/f7171b04a30361f5101edb2ff3c8d0fad190ea48#r411966586

Reply:
I've removed the `-t` which is unneeded and changed to using `IMAGE` as the variable name to make the meaning more clear.